src/services/types.ts

```diff
@@ -1379,6 +1386,12 @@ export interface CompletionEntry {
     isFromUncheckedFile?: true;
     isPackageJsonImport?: true;
     isImportStatementCompletion?: true;
+    /**
+     * For API purposes.
+     * Included for non-string completions only when `includeSymbol: true` option is passed to `getCompletionsAtPosition`.
+     * @example Get declaration of completion: `symbol.valueDeclaration`
+     */
+    symbol?: Symbol

```

Comment:
@andrewbranch All done, but decided to also add jsdoc on `symbol` property itself (since its always visible to user even if option is off). Anyway, don't see anything bad in extra documentation 😄